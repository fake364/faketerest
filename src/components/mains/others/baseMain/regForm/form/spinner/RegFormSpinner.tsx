import React from 'react';
import CircleSpinner, {
  SPINNER_COLORS
} from '../../../../../../../common/components/spinner/CircleSpinner';

type Props = {};

const RegFormSpinner: React.FC<Props> = () => {
  return (
    <div className="absolute top-0 left-0 w-full h-full bg-[rgba(0,0,0,0.5)] z-10 flex items-center justify-center">
      <CircleSpinner
        className={'w-[50px] h-[50px]'}
        color={SPINNER_COLORS.WHITE}
      />
    </div>
  );
};

export default RegFormSpinner;
