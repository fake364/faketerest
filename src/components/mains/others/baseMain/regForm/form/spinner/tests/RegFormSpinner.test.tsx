import React from 'react';
import { shallow } from 'enzyme';
import RegFormSpinner from '../RegFormSpinner';

describe('RegFormSpinner', () => {
  it('should render full screen spinner', () => {
    const wrapper = shallow(<RegFormSpinner />);
    expect(wrapper).toMatchSnapshot();
  });
});
