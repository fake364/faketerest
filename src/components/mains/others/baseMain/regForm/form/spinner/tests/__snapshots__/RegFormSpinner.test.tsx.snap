// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegFormSpinner should render full screen spinner 1`] = `
<div
  className="absolute top-0 left-0 w-full h-full bg-[rgba(0,0,0,0.5)] z-10 flex items-center justify-center"
>
  <CircleSpinner
    className="w-[50px] h-[50px]"
    color={1}
  />
</div>
`;
